templates/pc/README.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+
+This directory provides a RHEL6 based template, to support legacy guest OS
+which require an older pc-i440fx machine type to boot.
+Clusters that were deployed using the HyperConverged Operator do not include
+support for this machine type. To allow Virtual Machines to request this
+machine type, this option should be manually added.
+
+# Update HCO
+
+It is necessary to add the relevant machine type to a list of permitted types.
+This will allow Virtual Machines to request the pc-i440fx machine type.
+
+```
+$ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged \
+  kubevirt.kubevirt.io/jsonpatch='[{"op": "add", \
+    "path": "/spec/configuration/emulatedMachines", \
+    "value": ["q35*", "pc-q35*", "pc-i440fx-rhel7.6.0"] }]'
+```
+
+# OpenShift Virtualization 2.6.X
```

Comment:
Is this additional step on top of the previous section? Or is this an alternative path for <= 2.6?

Reply:
That's an alternative path for 2.6. I wasn't able to verify whether the previous step works as expected on 2.6.
I went over the existing documentation and it appears that config map is the suggested configuration option.

Reply:
```suggestion
# Update HCO (OpenShift Virtualization 2.6.X)
```

Reply:
@phoracek That's not HCO in this case.

Reply:
Oh of course, sorry

Reply:
Maybe, Update KubeVirt Config Map (OpenShift Virtualization 2.6.X)

Reply:
What about now?

Reply:
:+1: